### DISCUSSION TITLE
Is there anyway I can reduce the margin of toast wrapper?

### DISCUSSION THREAD

**Discussion by vicke4** [chakra-ui/chakra-ui#3504] [Q&A]
Hi,

I want to reduce the distance between adjacent toasts as it looks quite weird on the small screen I'm working on.

Any help would be appreciated. 

![custom chakra toast in a small screen](https://i.ibb.co/XCn7kHm/image.png)

**Comment by malashevskyi** [comment]
~~This may not be a good solution but you can change  it with global styles with scaling~~ [example on codesandbox](https://codesandbox.io/s/chakra-ui-change-toast-size-omtix?file=/src/App.js:91-224)

~~const theme = extendTheme({
&nbsp;&nbsp;styles: {
&nbsp;&nbsp;&nbsp;&nbsp;global: {
      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;".chakra-alert": {
        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;transform: "scale(1.15)"
      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;}
&nbsp;&nbsp;&nbsp;&nbsp;}
&nbsp;&nbsp;}
});~~
**Update: 24.03.2021**
Found the correct solution for how to do it:
```javascript
const Alert = {
  baseStyle: {
    container: {
      my: '-7px'
    }
  }
}
const theme = extendTheme({
  components: {
    Alert
  }
});
<ChakraProvider theme={theme}><App /></ChakraProvider>
```
I updated my previous [example on codesandbox](https://codesandbox.io/s/chakra-ui-change-toast-size-omtix?file=/src/App.js:91-224)

**Reply by malashevskyi** [reply]
Updated my answer

**Comment by chasinhues** [comment]
The spacing between the toasts is set here, so there's not going to be a nice way to do this at the moment.
https://github.com/chakra-ui/chakra-ui/blob/0de887f3b5110ad01ee7cf70e19bc759242da695/packages/toast/src/toast.tsx#L123

You could do something like this, though I'd avoid if it's not hugely necessary.
```css
.chakra-toast__inner {
    margin: 0.25rem !important;
}
```

**Comment by vicke4** [comment]
Thanks, @ChasinHues, @malashevskyi. I used both of your solutions to get the result. Targeting `.chakra-alert` was not working for me for some reason. Hence, I don't know which one to mark as an answer 😅
